### PR TITLE
updates changelog types; some updates for easier translation

### LINKED ISSUE #70923: New changelog categories missing from documentation
##### SUMMARY

Two new changelog categories, `breaking_changes`, and `security_fixes`, were added in #69968 but were not added to the [changelog fragment documentation](https://github.com/ansible/ansible/blob/4cc4cebc97ab1b9c5f0c424238d5352274b5dbad/docs/docsite/rst/community/development_process.rst#creating-a-changelog-fragment).

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME

`changelogs/config.yaml`
##### ANSIBLE VERSION

```paste below
2.11
```

##### CONFIGURATION

```paste below
default
```

##### OS / ENVIRONMENT

macOS 10.15.6
##### ADDITIONAL INFORMATION

Do we need to backport #69968 so we can backport security fixes to those branches and have the changelog build successfully?

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
Fixes #70923

Updates the community documentation to reflect the new changelog categories. This page likely needs more work to reflect the collections ecosystem, but this is a start.

Also related to #69968 and #70902 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com
changelogs
porting guides

### CODE REVIEW

File: docs/docsite/rst/community/development_process.rst [hunk 1]
```diff
@@ -152,23 +148,14 @@ We build short summary changelogs for minor releases as well as for major releas
...
 
-A single changelog fragment may contain multiple sections but most will only contain one section.
-The toplevel keys (bugfixes, major_changes, and so on) are defined in the
-`config file <https://github.com/ansible/ansible/blob/devel/changelogs/config.yaml>`_ for our release note tool. Here are the valid sections and a description of each:
+A single changelog fragment may contain multiple sections but most will only contain one section. The toplevel keys (bugfixes, major_changes, and so on) are defined in the `config file <https://github.com/ansible/ansible/blob/devel/changelogs/config.yaml>`_ for our release note tool. Here are the valid sections and a description of each:
+
+**breaking_changes**
+  Changes that break existing playbooks. This includes any changes to existing behavior that force users to update tasks. These entries are included in the :ref:`Ansible Porting Guides <porting_guides>`.
```
Comment (felixfontein) [inline docs/docsite/rst/community/development_process.rst]:
```suggestion
  Changes that break existing playbooks or roles. This includes any changes to existing behavior that force users to update tasks. These entries are included in the :ref:`Ansible Porting Guides <porting_guides>`.
```

File: docs/docsite/rst/community/development_process.rst [hunk 2]
```diff
@@ -152,23 +148,14 @@ We build short summary changelogs for minor releases as well as for major releas
...
 
 **major_changes**
   Major changes to Ansible itself. Generally does not include module or plugin changes.
```
Comment (felixfontein) [inline docs/docsite/rst/community/development_process.rst]: These are also included in the porting guide.

File: docs/docsite/rst/community/development_process.rst [hunk 3]
```diff
@@ -182,14 +169,16 @@ The toplevel keys (bugfixes, major_changes, and so on) are defined in the
 **removed_features**
   Features that were previously deprecated and are now removed.
```
Comment (felixfontein) [inline docs/docsite/rst/community/development_process.rst]: These are also included in the porting guide. (deprecated_features above too.)

**Comment by acozine** [comment]
Thanks @felixfontein for the comments, I incorporated them in c5da2d95e9fcfa1e0a2f9b3b5347993345bab7dc.

File: docs/docsite/rst/community/development_process.rst [hunk 4]
```diff
@@ -152,44 +148,37 @@ We build short summary changelogs for minor releases as well as for major releas
...
 **deprecated_features**
-  Features that have been deprecated and are scheduled for removal in a future release.
+  Features that have been deprecated and are scheduled for removal in a future release. Displayed in both the changelogs and the :ref:`Porting Guides <porting_guides>`.
 
 **removed_features**
-  Features that were previously deprecated and are now removed.
+  Features that were previously deprecated and are now removed. Displayed in both the changelogs and the :ref:`Porting Guides <porting_guides>`.
+
+**security_fixes**
+  Fixes that address CVEs or resolve security concerns. Include links to CVE information. Displayed in both the changelogs and the :ref:`Porting Guides <porting_guides>`.
```
Comment (felixfontein) [inline docs/docsite/rst/community/development_process.rst]:
```suggestion
  Fixes that address CVEs or resolve security concerns. Include links to CVE information.
```
These are not put into the porting guide (or only coincidently, when they result in a breaking change).

Comment (acozine) [reply docs/docsite/rst/community/development_process.rst]: Oh! TIL. Thanks, I'll fix that.

**Review by felixfontein — APPROVED** [review]
LGTM

**Review by samccann — APPROVED** [review]

**Pull request merged**